src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationlocation/Example2.java

```diff
@@ -15,17 +15,23 @@
 
 package com.puppycrawl.tools.checkstyle.checks.annotation.annotationlocation;
 
+import org.antlr.v4.runtime.misc.NotNull;
```

Comment:
This is a deprecated annotation (and uncommon in use as well), can we use a more common annotation such as `javax.annotation.Nullable`?

Reply:
Yes, better to use well known annotations that most users know.

Reply:
Valid point. I was scratching the IDE and i javax.annotation.Nonnull can be used replacing `@NotNull` with `@Nonnull`. So, now it would look like something below:
```
  @Nonnull
  private boolean field2;
```

Is it acceptable because i don't see any change in the functionality?

Reply:
Yes, this is good.

Reply:
done. @romani, can we add hacktoberfest-accepted label to this PR?